contents/blog/posthog-vs-logrocket.md

```diff
@@ -0,0 +1,750 @@
+---
+date: 2023-03-03
+title: PostHog vs Logrocket
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/posthog-vs-logrocket/posthog-vs-mixpanel.jpeg
+featuredImageType: full
+author:
+    - joe-martin
+category: General
+tags:
+    - Comparisons
+    - Guides
+---
+
+LogRocket is a frontend monitoring platform which, like PostHog, combines multiple tools into one platform. In this article we’ll explain some of the biggest similarities and differences between PostHog and LogRocket, but here’s the short version:
+
+-   LogRocket focuses heavily on helping frontend developers to detect and solve issues. It offers tools such as session recording, analytics and performance monitoring.
+
+-   PostHog is an all-in-one platform that helps teams build better products. It includes many of the same tools as LogRocket, including session recording, analytics and performance monitoring — but also extends to feature flags, A/B experiments and more.
+
+In this article we’ll dive deeper into these differences and will:
+
+-   Compare PostHog and LogRocket’s core features
+-   Examine PostHog and LogRocket’s library support
+-   Discuss what integrations are available with other tools
+-   Explore other frequently asked questions about PostHog and LogRocket
+
+## How do PostHog and LogRocket differ?
```

Comment:
Feel free to tweak, but I like to get these sections to tight single paragraphs.